cas-server-integration-hazelcast/src/main/java/org/jasig/cas/ticket/registry/HazelcastTicketRegistry.java

```diff
@@ -151,8 +151,32 @@ private void deleteChildren(final TicketGrantingTicket ticket) {
             }
         }
     }
+    
+    @Override
+    public int sessionCount() {
+        int count = 0;
+        final Collection<Ticket> collection = getTickets();
```

Comment:
Ah, got ya. I was also confused about int return type vs long. It's 4.2. OK, nothing to see here. Carry on :-)